Gemfile.lock

```diff
@@ -0,0 +1,261 @@
+GEM
```

Comment:
Once you're at it, could you please check the security warning that GitHub displays at the repo front page?

Reply:
This should fix the security warning. The Gemfile.lock sets the nokogiri version to 1.8.2